### DISCUSSION TITLE
can not find symbol PROCESS_MITIGATION_REDIRECTION_TRUST_POLICY

### DISCUSSION THREAD

**Discussion by chenghuiwen** [winsiderss/systeminformer#1137] [Questions / Answers] [answered]
when I try to build ProcessHacker.sln, VS hint me can not find symbol PROCESS_MITIGATION_REDIRECTION_TRUST_POLICY.
How I solve this problem.

**Comment by Biswa96** [comment, marked as answer]
Use newer Windows SDK version 10.0.22000.0 or higher.

**Reply by chenghuiwen** [reply]
It works，Thanks a lot
